doc/modules/outlier_detection.rst

```diff
@@ -150,12 +150,29 @@ implementation. The :math:`\nu` parameter, also known as the margin of
 the One-Class SVM, corresponds to the probability of finding a new,
 but regular, observation outside the frontier.
 
+The Support Vector Data Description (:ref:`svm_svdd`) is an alternative
+model for estimating the support of a data distribution. It was proposed
+by Tax and Duin, and later reformulated by Chang et al. The reparametrized
+SVDD model, which has better parameter interpretability, is implemented
+in the :class:`svm.SVDD` object in the  :ref:`svm` module. The interface
+as well as the interpretation of the parameters is similar to the
+:ref:`svm_one_class_svm` model.
+
```

Comment:
Could you please update the `examples/plot_anomaly_comparison.py` to add a new column for SVDD next to the one class SVM column? This would make it more visible as this example is included at the top of this section.